packages/react-select/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-select",
-  "version": "4.3.0",
```

Comment:
Version should not be changed by hand. This is handled through the release process.